lib/screens/user_registration/widgets/username_input.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+///For Input of User Name
+class UserNameInput extends StatelessWidget {
```

Comment:
if the widget is quite small, you don't need to create a separate file for it
integrate it in the main screen file directly

Reply:
Fixed